### ISSUE TITLE
The picker_loss decreases during training, but the F1 index does not increase

### ISSUE THREAD

**Issue by EternityZY** [smousavi05/EQTransformer#15]
Thank you very much for your work, he is instructive to me
I have a problem here. I want to fully reproduce your work. I use the complete Stanford EArthquake Dataset (STEAD) and use the hyperparameters in the Example to train the model in the repo. 
However, during the training process, I found that the overall Loss is decreasing(Including p_picker_loss and s_picker_loss), while the F1 index has been declining (p_picker_f1 and s_picker_f1), even with the decrease of Loss, the F1 index has also decreased.
By the way, detector_f1 will increase as detector_loss decreases.
I haven't changed any code or parameters. What is the problem?

Such as:
 loss: 0.0989 - detector_loss: 0.2477 - picker_P_loss: 0.0592 - picker_S_loss: 0.0643 - 
                       detector_f1: 0.7590 - picker_P_f1: 3.2014e-04 - picker_S_f1: 3.5400e-04

**Comment by smousavi05** [comment]
Those hyperparameters in the example notebook are only for the demonstration and building a simple network. Could you try one more time with the default parameters in the trainer module? moreover, could you tell me at which epoch you see this problem? is it at the beginning of the training or toward the end?

**Comment by EternityZY** [comment]
Thank you for your reply, I will try to train again using the default parameters. 
I found this problem at the beginning of training, and the results in the late training are as follows：
loss: 0.0061 -  picker_P_f1: 0.4324 - picker_S_f1: 0.2738

**Comment by smousavi05** [comment]
So then I would also suggest you continue the training. The network that we trained had 5 CNN and 3 LSTM residual blocks and have been trained for almost 80 epochs using the full dataset. However, you should get similar results even if you use a smaller training size (i.e. ~ 20,000) providing you give it enough time for the training.

**Comment by EternityZY** [comment]
Thank you very much for your help. Your work is very instructive for me.

In addition, I would like to know what level of F1 index of P-wave and S-wave is maintained during your training?

**Comment by smousavi05** [comment]
F1 for P-wave was ~0.95 and for S-wave ~0.7
